shells/fish/Portfile

```diff
@@ -29,10 +29,7 @@ depends_lib             port:libiconv \
                         port:ncurses \
                         port:gettext
```

Comment:
`gettext` dependencies have changed:

Instead of:
```
depends_lib port:gettext
```

You need to add `gettext` for `build` dependencies, and `gettext-runtime` for `lib` dependencies.  So this would look like:

```
depends_build-append    port:gettext

depends_lib-append      port:libiconv \
                        port:ncurses \
                        port:gettext-runtime
```